src/javascripts/app.js

```diff
@@ -1,3 +1,3 @@
 import './modules'
 
-console.log(`app.js has loaded!`)
```

Comment:
Agreed, my mistake.  I thought backticks were "stage-1" (which I had removed because it was causing build errors, and using draft features in prod isn't always appropriate), but looks like they're also valid in "es2015".